sdk/identity/azure-identity/test/simplified_header.cpp

```diff
@@ -8,6 +8,12 @@
  *
  */
 
+// Verify that including windows.h, which defines "min" and "max" as macros, does not fail any code
+// in SDK headers.
+#if defined(_MSC_VER)
+#include <windows.h>
+#endif // _MSC_VER
```

Comment:
I'm concerned that we need to perform this same test for every SDK in the repo. Storage and keyvault, for example, have been omitted here. Is there any way we can guarantee future coverage without needing to propagate an identical test case to each SDK as it's added?

Reply:
This would've been a way to go, in my opinion - ideally, each lib would have its own simplified header smoketest, just as core and identity currently do. So, it would be cloned and adapted for each new library. That is the reason I did not add windows.h inclusion test to storage libs - because storage currently lacks the simplified header test.